include/nccl_ofi_param.h

```diff
@@ -234,9 +234,9 @@ OFI_NCCL_PARAM_INT(disable_gdr_required_check, "DISABLE_GDR_REQUIRED_CHECK", 0);
 OFI_NCCL_PARAM_INT(disable_dmabuf, "DISABLE_DMABUF", 0);
 
 /*
- * Maximum size of a message in bytes before message is multiplexed
+ * Messages sized larger than this threshold will be striped across multiple rails
  */
-OFI_NCCL_PARAM_UINT(round_robin_threshold, "ROUND_ROBIN_THRESHOLD", (256 * 1024));
+OFI_NCCL_PARAM_UINT(min_stripe_size, "MIN_STRIPE_SIZE", (256 * 1024));
```

Comment:
Is 256K the right `MIN_STRIPE_SIZE`? 
This would mean multiplexing with 4 rail will happens at 1M, which is different than what we did before. 
@arunkarthik-akkart is investigating this.